mkosi.extra/usr/lib/systemd/system-preset/10-particleos.preset

```diff
@@ -25,8 +25,8 @@ enable power-profiles-daemon.service
 # Our own service to run systemctl preset --global.
 enable preset-global.service
 
-# Should be enabled manually by the user after first boot.
-disable gdm.service
+enable gdm.service
```

Comment:
please also add gdm3.service, as that's how it's called on debian, ubuntu and derivatives